cfme/web_ui/paginator.py

```diff
@@ -78,28 +86,28 @@ def sort_by(sort):
     sel.select(Select(select), sel.ByText(sort))
 
 
+def _records():
+    matches = re.search(_records_re, _page_nums()).groupdict()
+    return {k: int(v) for k, v in matches.iteritems()}
+
+
 def rec_offset():
     """ Returns the first record offset."""
-    offset = re.search('\((Item|Items)*\s*(\d+)', _page_nums())
-    return offset.groups()[1]
+    return _records()['offset']
 
 
 def rec_end():
     """ Returns the record set index."""
-    offset = re.search('-(\d+)', _page_nums())
-    if offset:
-        return offset.groups()[0]
+    end = _records().get('end')
+    if end:
+        return end
```

Comment:
In the far reaches of my mind I want to say yes, but I can't give you a concrete example :(